Description: Build against HDF4ALT on Debian.
Origin: http://bugs.debian.orf/655869
Forwarded: not-needed
Last-UpdatedL: 2012-09-19

Index: grads-2.2.1/configure.ac
===================================================================
--- grads-2.2.1.orig/configure.ac
+++ grads-2.2.1/configure.ac
@@ -675,11 +675,11 @@ if test "Z$ga_supplib_dir" != "Z" ; then
     echo looking in $ga_supplib_dir 
     GA_SET_FLAGS([])
     AC_CHECK_HEADER([mfhdf.h],
-    [ AC_CHECK_LIB([df], [main], 
-      [ AC_CHECK_LIB([mfhdf], [main],
+    [ AC_CHECK_LIB([dfalt], [main], 
+      [ AC_CHECK_LIB([mfhdfalt -ldfalt], [main],
         [ use_hdf=yes 
 #          GA_SET_LIB_VAR([hdf_libs], [mfhdf df jpeg z udunits])
-          GA_SET_DYNLIB_VAR([hdf_libs], [mfhdf df jpeg z udunits])
+          GA_SET_DYNLIB_VAR([hdf_libs], [mfhdfalt dfalt jpeg z udunits])
         ])
       ])
     ])
Index: grads-2.2.1/m4/hdf4.m4
===================================================================
--- grads-2.2.1.orig/m4/hdf4.m4
+++ grads-2.2.1/m4/hdf4.m4
@@ -189,11 +189,11 @@ dnl nor required everywhere
   ac_hdf4_lib='no'
   AC_CHECK_LIB_NOCACHE_HDF4([z],[deflate],
   [ AC_CHECK_LIB_NOCACHE_HDF4([jpeg],[jpeg_start_compress],
-    [ AC_CHECK_LIB_NOCACHE_HDF4([df],[Hopen],
-      [ AC_CHECK_LIB_NOCACHE_HDF4([mfhdf],[SDstart],
+    [ AC_CHECK_LIB_NOCACHE_HDF4([dfalt],[Hopen],
+      [ AC_CHECK_LIB_NOCACHE_HDF4([mfhdfalt],[SDstart],
         [ ac_hdf4_lib="yes"
-          HDF4_LIBS="-lmfhdf -ldf -ljpeg -lz $HDF4_LIBS"
-        ],[],[-ldf -ljpeg -lz])
+          HDF4_LIBS="-lmfhdfalt -ldfalt -ljpeg -lz $HDF4_LIBS"
+        ],[],[-ldfalt -ljpeg -lz])
       ],[],[-ljpeg -lz])
     ])
   ])
