Description: Declaring static breaks access from gagui.c
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2017-10-20
Forwarded: no

Index: grads-2.2.1/src/grads.c
===================================================================
--- grads-2.2.1.orig/src/grads.c
+++ grads-2.2.1/src/grads.c
@@ -48,7 +48,7 @@ void write_command_log(char *logfile);
 
 static struct gaupb *upba=NULL;     /* Anchor for user defined plug-in */
 char *gxgnam(char *);               /* This is also in gx.h */
-static struct gacmn gcmn;  
+struct gacmn gcmn;  
 static struct gawgds wgds; 
 extern struct gamfcmn mfcmn;
 
